packages/server/graphql/intranetSchema/mutations/draftEnterpriseInvoice.ts

```diff
@@ -120,7 +120,10 @@ export default {
     try {
       user = await getBillingLeaderUser(email, orgId, dataLoader)
     } catch (e) {
-      return {error: {message: e.message}}
+      if (e instanceof Error) {
+        return {error: {message: e.message}}
+      }
+      return {error: {message: 'Unable to get billing leader user'}}
```

Comment:
Not a big deal and probably depends on personal preferences but this could be simplified to 
```
      return {
        error: {message: e instanceof Error ? e.message : 'Unable to get billing leader user'}
      }
```
or 
```
      const message = e instanceof Error ? e.message : 'Unable to get billing leader user'
      return {
        error: {message}
      }
```

I don't have any strong preferences here so feel free to do whatever you think is best 👍🏻